service/src/main/java/greencity/service/ubs/OrderBagService.java

```diff
@@ -0,0 +1,94 @@
+package greencity.service.ubs;
+
+import greencity.entity.order.Bag;
+import greencity.entity.order.OrderBag;
+import greencity.exceptions.NotFoundException;
+import greencity.repository.OrderBagRepository;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static greencity.constant.ErrorMessage.BAG_NOT_FOUND;
+
+@Service
+@AllArgsConstructor
```

Comment:
You can remove this annotation since the Data annotation already contains RequiredArgsConstructor, and the bean should initialized by the constructor.

Reply:
fixed